comps/lvms/tgi-llava/lvm_tgi.py

```diff
@@ -46,6 +48,11 @@ async def lvm(request: Union[LVMDoc, LVMSearchedMultimodalDoc]) -> TextDoc:
         if logflag:
             logger.info("[LVMSearchedMultimodalDoc ] input from retriever microservice")
         retrieved_metadatas = request.metadata
+        if retrieved_metadatas is None or len(retrieved_metadatas) == 0:
```

Comment:
`if not retrieved_metadatas:` should be more concise

Reply:
Thank for your feedback. @Spycsh 
I have updated it to be `if not retrieved_metadatas:`